common-utils/src/main/java/eu/arrowhead/common/intf/properties/validators/HttpOperationsValidator.java

```diff
@@ -1,21 +1,82 @@
 package eu.arrowhead.common.intf.properties.validators;
 
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.springframework.stereotype.Service;
 
+import eu.arrowhead.common.Utilities;
 import eu.arrowhead.common.exception.InvalidParameterException;
+import eu.arrowhead.common.http.model.HttpOperationModel;
 import eu.arrowhead.common.intf.properties.IPropertyValidator;
 
 @Service
 public class HttpOperationsValidator implements IPropertyValidator {
 
+	//=================================================================================================
+	// members
+
+	private final Logger logger = LogManager.getLogger(getClass());
+
 	//=================================================================================================
 	// methods
 
 	//-------------------------------------------------------------------------------------------------
+	// propertyValue should be a map with string keys (operation name) and HttpOperationModel value
+	// at least one pair should be in the map
+	// an HttpOperationModel object should contains a not null method and a non-blank path
+	// no need for normalization
```

Comment:
Why not? Lowercasing and trimming the operation names and also trimming the paths seems reasonable to me. Maybe uppercasing the methods to match with the enums?

Reply:
You are right. I've done the operation and path normalization. However, the method is not a string, it is already an HttpMethod => according our Skype meeting I will change to string and adjust the validator.